crypto/fipsmodule/cipher/cipher.c

```diff
@@ -70,6 +70,7 @@
 
 void EVP_CIPHER_CTX_init(EVP_CIPHER_CTX *ctx) {
   OPENSSL_memset(ctx, 0, sizeof(EVP_CIPHER_CTX));
+  ctx->poisoned = 1;
```

Comment:
`ctx->poisoned` also needs to be set in `EVP_CIPHER_CTX_new` and `EVP_CIPHER_CTX_cleanup` right below this.